pkg/repository/clone_mirror.go

```diff
@@ -309,7 +310,11 @@ func cloneComponents(repo *V1Repository,
 					continue
 				}
 				if err := download(targetDir, tmpDir, repo, &versionItem); err != nil {
-					return nil, errors.Annotatef(err, "download resource: %s", name)
+					if strings.Contains(err.Error(), "not found") {
```

Comment:
I think it's better check for HTTP status code, that might need to be implemented in underlying functions.